lib/src/main/java/edu/stanford/futuredata/macrobase/analysis/summary/apriori/ExplanationResult.java

```diff
@@ -0,0 +1,62 @@
+package edu.stanford.futuredata.macrobase.analysis.summary.apriori;
+
+
+import java.util.Map;
+import java.util.StringJoiner;
+
+public class ExplanationResult {
+    private Map<String, String> matcher;
+    private double matchedOutlier, matchedCount;
```

Comment:
Having these raw counts makes things much easier for users to interpret (How many points is this? vs What is a "support" fraction?) Also lets us add new measures easily since we have these raw counts.